python/dbsp-api-client/dbsp_api_client/models/error_response.py

```diff
@@ -11,12 +11,13 @@
 
 @attr.s(auto_attribs=True)
 class ErrorResponse:
-    """Pipeline manager error response.
+    """Information returned by REST API endpoints on error.
```

Comment:
Should this change be in this PR?

Reply:
It's a leftover from a previous PR, which was already merged. But it's a separate commit.